### PR TITLE
Feat: Product list page

### PR DESCRIPTION
**Pull request by sri-rishi**
#### Files to check

- ProductList.jsx
- ProdcutDataList.jsx
- VerticalCard.jsx
- Sidebar.jsx
- Button.jsx
- data folder

### CODE REVIEW

File: src/data/index.js [hunk 1]
```diff
@@ -2,5 +2,6 @@ import { brandLogoData } from "./brandCategoryCover.data";
 import { categorySectionItems } from "./categoryCover.data";
 import { interestCoverData } from "./interestCategoryCover.data";
 import { newArrivalCover } from "./newCategoryCover.data";
+import { ratingData } from "./rating.data";
 
-export {brandLogoData, categorySectionItems, interestCoverData, newArrivalCover}
\ No newline at end of file
+export {brandLogoData, categorySectionItems, interestCoverData, newArrivalCover, ratingData};
```
Comment (bhtibrewal) [inline src/data/index.js:7]: Good work, mantaining separate data

File: src/data/rating.data.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export const ratingData = [
```
Comment (bhtibrewal) [inline src/data/rating.data.js:1]: Good work thinking in term or making things dynamic

File: src/pages/ProductList/components/SideBar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
+
+import { Button } from "../../../../Components/index"
+import { brandLogoData, categorySectionItems, interestCoverData, ratingData } from "../../../../data"
+
+export const SideBar = () => {
+    return (
+        <aside className="aside-box grid">
```
Comment (bhtibrewal) [inline src/pages/ProductList/components/SideBar/Sidebar.jsx:7]: Break down this page and try to make them resuable

File: src/Components/VerticalCard/VerticalCard.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { AiFillHeart, FaShoppingCart} from "../../assets/icons";
+import { Button } from "../index";
+
+
+export const VerticalCard = (props) => {
+    const { brandName,categoryName,imageSrc,inStock,fastDelivery,newArrival,commingSoon,interestCategory,price,ratings,description} = props.productItems
```
Comment (git-0r) [inline src/Components/VerticalCard/VerticalCard.jsx:6]: suggestion: directly destructure productItems.

**Review by git-0r — APPROVED** [review]

File: src/pages/ProductList/components/SideBar/Sidebar.jsx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                </div>
+                <input className="m-vl-1 filter-items" type="range" min="1" max="1000" value="1000" id="myRange" />
+            </div>
+            <div className="filter-sections m-hr-1 flex-column">
+                <div className="flex-row align-center justify-between">
+                    <span className="sub-filt-head font-weight-5">Sort</span>
+                </div>
+                <ul className="m-vl-1 filter-items">
+                    <li className="align-center">
+                        <label htmlFor="price-l-to-h">
```
Comment (Naman-Saxena1) [inline src/pages/ProductList/components/SideBar/Sidebar.jsx:24]: Can make label tag htmlFor and input tag id more descriptive for readability.

File: src/pages/ProductList/components/SideBar/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+                            <input className="filter-inputs" type="radio" id="price-l-to-h" name="sort" aria-label="Price Low to High" />
+                            Price - Low to High
+                        </label>
+                    </li>
+                    <li>
+                        <label htmlFor="price-h-to-l">
+                            <input className="filter-inputs" id="price-h-to-l" type="radio" name="sort"  aria-label="Price High to Low" />
+                            Price - High to Low
+                        </label>
```
Comment (Naman-Saxena1) [inline src/pages/ProductList/components/SideBar/Sidebar.jsx:33]: Same, naming conventions can be improved.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
I suggested few things.
Code looked fine to me, can't find any issues. 👍 
I understand that there is preview link but it would be great if you could also add screenshots in description.

**Review by bhtibrewal — APPROVED** [review]
Good work here, okay to merge

**Pull request merged**